com.unity.visualeffectgraph/Editor/Models/VFXGraph.cs

```diff
@@ -21,19 +21,59 @@ namespace UnityEditor.VFX
     [InitializeOnLoad]
     class VFXGraphPreprocessor : AssetPostprocessor
     {
-        void OnPreprocessAsset()
+        static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths)
         {
-            bool isVFX = VisualEffectAssetModicationProcessor.HasVFXExtension(assetPath);
-            if (isVFX)
+            List<string> assetToReimport = null;
+
+            foreach (var assetPath in importedAssets)
             {
-                VisualEffectResource resource = VisualEffectResource.GetResourceAtPath(assetPath);
-                if (resource == null)
-                    return;
-                VFXGraph graph = resource.graph as VFXGraph;
-                if (graph != null)
-                    graph.SanitizeForImport();
-                else
-                    Debug.LogError("VisualEffectGraphResource without graph");
+                bool isVFX = VisualEffectAssetModicationProcessor.HasVFXExtension(assetPath);
+                if (isVFX)
+                {
+                    VisualEffectResource resource = VisualEffectResource.GetResourceAtPath(assetPath);
+                    if (resource == null)
+                        return;
```

Comment:
This `return;` should be `continue;` but vfx resources without vfxgraph doesn't happen anymore

Reply:
Resolved at https://github.com/Unity-Technologies/Graphics/pull/4971/commits/8c442a8cf3d006f8406e31e571075ac00d8f271c (cc @julienf-unity just in case)